src/components/editor/EditAuthorsForm.jsx

```diff
@@ -1,80 +1,37 @@
 import React from 'react';
+import ReactDOM from 'react-dom';
 import BaseComponent from 'lib/BaseComponent';
 import _ from 'lodash';
 
 export default class EditAuthorsForm extends BaseComponent {
   constructor(props) {
     super(props);
     this.safeSetState({
-      // Key is slug, value is name
-      authorsAdded: {},
-      // An object where key is slug and value is whether it's deleted or not
-      authorsDeleted: _.reduce(this.props.data, (cur, next) => {
-        cur[next.slug] = false;
-        return cur;
-      }, {})
+      addAuthorValue: ""
     });
     this.handleAddAuthor = this.handleAddAuthor.bind(this);
   }
 
   componentDidUpdate() {
-    this.props.onChange(this.state.authorsAdded, this.state.authorsDeleted);
+    this.props.onChange(ReactDOM.findDOMNode(this).parentNode);
```

Comment:
Yeah but reading your code it looks like you don't use this formNode argument?

in general dom manipulations like this should ring red flags. It makes our component too fragile and future refactors could easily break it